### PR TITLE
add quick fix logic

### PR DESCRIPTION
**Pull request by poopoothegorilla**

### CODE REVIEW

File: deployment/manifest_step.go [hunk 1]
```diff
@@ -208,11 +208,63 @@ func (s *ManifestStep) Destroy() error {
...
 }
+
+func deleteRC(namespace string, meta *api.ObjectMeta) error {
+	// SCALE RC DOWN TO 0
+	rc, err := client.ReplicationControllers(namespace).Get(meta.Name)
+	if err != nil {
+		return err
+	}
+	var i int32
+	rc.Spec.Replicas = &i
```
Comment (therealplato) [inline deployment/manifest_step.go]: what's this for?

Comment (poopoothegorilla) [inline deployment/manifest_step.go]: set replicas to 0

Comment (poopoothegorilla) [inline deployment/manifest_step.go]: which needs to be *int32 type

File: deployment/manifest_step.go [hunk 2]
```diff
@@ -208,11 +208,63 @@ func (s *ManifestStep) Destroy() error {
...
+	client.ReplicationControllers(namespace).Update(rc)
+	time.Sleep(1 * time.Second) // Wait for Kubernetes to delete pods
+	rc, err = client.ReplicationControllers(namespace).Get(meta.Name)
+	if err != nil {
+		return err
+	}
+	if rc.Status.Replicas != 0 {
+		return errors.New("deployment: RC deletion not successful")
+	}
+	// WATCH REPLICATION CONTROLLER
```
Comment (therealplato) [inline deployment/manifest_step.go]: will we use this comment in the future?

Comment (poopoothegorilla) [inline deployment/manifest_step.go]: i think it is better

Comment (poopoothegorilla) [inline deployment/manifest_step.go]: the sleep is alittle hacking in my opinion but it matches our logic in updating rcs

**Comment by therealplato** [comment]
+1 although i would consider a comment as to why we are using the default int16 pointer value

**Comment by poopoothegorilla** [comment]
@therealplato addressed suggestion

File: deployment/manifest_step.go (hunk 2 above)
Comment (rickyxiv) [inline deployment/manifest_step.go]: Lets bump it to 10 seconds this should ensure the pod can terminate cleanly

**Comment by rickyxiv** [comment]
👍

**Pull request merged**